website/source/docs/providers/aws/r/db_option_group.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "aws"
+page_title: "AWS: aws_db_option_group"
+sidebar_current: "docs-aws-resource-db-option-group"
+---
+
+# aws\_db\_option\_group
+
+Provides an RDS DB option group resource.
+
+## Example Usage
+
+```
+resource "aws_db_option_group" "bar" {
+	option_group_name = "option-group-test-terraform"
+	option_group_description = "Terraform Option Group"
+	engine_name = "sqlserver-ee"
+	major_engine_version = "11.00"
+
+	option {
+		option_name = "mirroring"
+	}
+
+	option {
+		option_name = "TDE"
+	}
+	
+	apply_immediately = true
+}
```

Comment:
hclfmt here por favor - essentially two-space indent / align equals in blocks. :+1: